packages/cli-exec/src/exec.js

```diff
@@ -92,7 +92,14 @@ export const exec = command('exec', {
   await percy?.stop(!!error);
 
   // forward any returned status code
-  if (status) exit(status, error);
+  if (status) {
+    exit(status, error);
+  } else {
+    // force exit post timeout
+    setTimeout(function() {
+      process.exit(0);
+    }, 10000);
+  }
```

Comment:
better promisify and await on sleep time of 10 secs. Then we can do standard process.exit

like 
```
await 10 sec 
process.exit(0)
```

also dont need to put it in `else`